### ISSUE TITLE
Indent behaves inconsistently for single and multiple variable declartions

### ISSUE THREAD

**Issue by zuzusik** [eslint/eslint#10551]
**Tell us about your environment**

* **ESLint Version: 5.0.1**
* **Node Version: 8.11.1**
* **npm Version: 5.6.0**

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "env": {
    "es6": true
  },
  "parserOptions": {
    "sourceType": "module"
  },
  "rules": {
    "indent": ["error", 2, {
      "VariableDeclarator": {"var": 2, "let": 2, "const": 3}
    }]
  }
}

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
let some = {
      a: 1
    };
```

```bash
./node_modules/eslint/bin/eslint.js 1.js
```

**What did you expect to happen?**

no errors

**What actually happened? Please include the actual, raw output from ESLint.**
```
  2:1  error  Expected indentation of 2 spaces but found 6  indent
  3:1  error  Expected indentation of 0 spaces but found 4  indent

✖ 2 problems (2 errors, 0 warnings)
  2 errors, 0 warnings potentially fixable with the `--fix` option.
```

**And the following code passes well:**
```js
let some = {
      a: 1
    },
    other = 123;
```
This is inconsistent and brings inconvenience as we are forced to change indentation of the first `let` when second is added.

**Comment by not-an-aardvark** [comment]
Hi, thanks for the report. This is a known behavior and is working as intended -- also see https://github.com/eslint/eslint/issues/3352.

**Comment by zuzusik** [comment]
@not-an-aardvark why not giving people option to treat one var exactly the same as list of vars? how it is done now **forces** the inconsistency. I don't feel good about it and looks like a lot of people from #3352 are on the same page with me.

**Comment by zuzusik** [comment]
@not-an-aardvark I was able to achieve desired behavior with just changing [this line](https://github.com/eslint/eslint/blob/2e68be6/lib/rules/indent.js#L1317) to following:

```
if (options.VariableDeclarator.treatOneVarLikeList || node.declarations[node.declarations.length - 1].loc.start.line > node.loc.start.line) {
```
and adding treatOneVarLikeList to schema

So it is a tiny code change, which is just about adding one boolean option and it is not a breaking change

Would you accept a PR for it? 

If yes - I need an advice on naming of the option - not sure if `treatOneVarLikeList` works well

**Comment by not-an-aardvark** [comment]
Personally I would be mildly opposed to adding an option -- the `indent` rule is already difficult to maintain due to the complexity and the number of options, so I don't think we should add additional options except if they would benefit a very large percentage of users. People have a lot of stylistic preferences, and unfortunately we can't support all of them. However, if there are other team members that think this is a worthwhile change, I won't block it.
